dali/pipeline/operators/reader/loader/indexed_file_loader.h

```diff
@@ -38,7 +38,9 @@ class IndexedFileLoader : public Loader<CPUBackend, Tensor<CPUBackend>> {
     }
 
   void ReadSample(Tensor<CPUBackend>* tensor) override {
-    if (current_index_ == indices_.size()) {
+    if ((current_index_ == indices_.size()) ||
```

Comment:
extract this so that it can be reused in both loaders?

Reply:
Done